python/ray/_private/runtime_env/agent/runtime_env_agent.py

```diff
@@ -201,7 +202,10 @@ def __init__(
         # Maps a serialized runtime env to a lock that is used
         # to prevent multiple concurrent installs of the same env.
         self._env_locks: Dict[str, asyncio.Lock] = dict()
-        self._gcs_aio_client = GcsAioClient(address=self._gcs_address)
+        self._gcs_aio_client = GcsAioClient(
+            address=self._gcs_address,
+            nums_reconnect_retry=ray._config.gcs_rpc_server_reconnect_timeout_s(),
```

Comment:
I don't think we need this change. The timeout for OldGcsAioClient is already `RAY_py_gcs_connect_timeout_s`